### DISCUSSION TITLE
setup method is not available

### DISCUSSION THREAD

**Discussion by zh199609** [seleniumbase/SeleniumBase#2644] [General]
class FailingTests(BaseCase):

    def setup(self):
        print('this is setup')
    def test_xxx(self):
        pass

Run As：
```python
if __name__ == '__main__':
    pytest.main(['my_first_test.py', '-v', '--rs'])
``` 

The setup method in pytest was not executed，Is it not supported

Need to be written as setUp ?

**Comment by mdmintz** [comment]
See [examples/boilerplates/base_test_case.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/boilerplates/base_test_case.py) for an example of customizing  `setUp()` and `tearDown()` properly.
